Module.cs

```diff
@@ -128,7 +128,22 @@ private async Task LoadRoles()
                 {
                     try
                     {
-                        role.Icon = ContentsManager.GetTexture(role.IconPath);
+                        if (role.IconPath.StartsWith("icons"))
+                        {
+                            role.Icon = ContentsManager.GetTexture(role.IconPath);
+                        }
+                        else
```

Comment:
Idea: in the future, maybe we could add a "squadtracker" prefix so it looks for the image in the config folder, instead of giving a full path!